From f99db8c83cc3958f456d1d736f5494bef3abd074 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 29 Dec 2022 20:51:29 +0200
Subject: [PATCH] [SW-0] memory: reuse sizes.h for region sizes

Reuse defines from sizes.h instead of open coding hex numbers and adding
comments.

Change-Id: Ife9386bfd1ab564a518a4c17a9b3dd5d2ec16ccd
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/include/memory.h b/include/memory.h
index 3623d7c..f5711b9 100644
--- a/include/memory.h
+++ b/include/memory.h
@@ -8,19 +8,20 @@
 
 #include <stdint.h>
 #include <types.h>
+#include <sizes.h>
 
 #include <common.h>
 
-#define MEM_GP_BASE		(SDRAM_BASE + 0x40000)	/* SRAM_BASE + 256K */
-#define MEM_GP_SZ		0x80000			/* 512K */
-#define MEM_BOOT_INFO_BASE	(SDRAM_BASE + 0xC0000)	/* SRAM_BASE + 768K */
-#define MEM_BOOT_INFO_SZ	0x40000			/* 256K */
-#define MEM_LKD_PCI_BASE	(SDRAM_BASE + 0x200000)	/* SRAM_BASE + 2M */
-#define MEM_LKD_PCI_SZ		0x200000		/* 2M */
-#define MEM_FWUPD_PCI_BASE	(SDRAM_BASE + 0x600000)	/* SRAM_BASE + 6M */
-#define MEM_FWUPD_PCI_SZ	0x200000		/* 2M */
-#define MEM_I2C_BASE		(SDRAM_BASE + 0x800000)	/* SRAM_BASE + 8M */
-#define MEM_I2C_SZ		0x200000		/* 2M */
+#define MEM_GP_BASE		(SDRAM_BASE + SZ_256K)
+#define MEM_GP_SZ		SZ_512K
+#define MEM_BOOT_INFO_BASE	(SDRAM_BASE + SZ_512K + SZ_256K)
+#define MEM_BOOT_INFO_SZ	SZ_256K
+#define MEM_LKD_PCI_BASE	(SDRAM_BASE + SZ_2M)
+#define MEM_LKD_PCI_SZ		SZ_2M
+#define MEM_FWUPD_PCI_BASE	(SDRAM_BASE + SZ_4M + SZ_2M)
+#define MEM_FWUPD_PCI_SZ	SZ_2M
+#define MEM_I2C_BASE		(SDRAM_BASE + SZ_8M)
+#define MEM_I2C_SZ		SZ_2M
 
 #if defined(SRAM_BIST_DATA_BASE)
 #define MEM_SRAM_BIST_DATA_BASE	SRAM_BIST_DATA_BASE	/* Addr to store SRAM BIST data in ppboot */
